gef.py

```diff
@@ -750,12 +750,14 @@ def __init__(self, addr, name=None):
         except:
             self.__arena = MallocStateStruct(addr)
             self.__addr = self.__arena.addr
-        finally:
+        try:
             self.top             = int(self.top)
             self.last_remainder  = int(self.last_remainder)
```

Comment:
I am realizing that we are shadowing the last_remainder and other fields from the GDB arena to our own. I wonder whether the GDB one changes behind the scenes and this is causing issues.

Reply:
yes, those fields are all calculated by GDB, not GEF. so a change over there could potential break GEFs heap commands. But I don't think that has happened lately. Of course it would be cool to do this ourselves and therefore not be dependent on GDB for that but on the other side I don't know if it is worth the effort when GDB already does it :)